lib/services/application.dart

```diff
@@ -161,114 +147,23 @@ class _LinuxApplicationService extends ApplicationService {
   }
 }
 
-class _BuiltInApplicationService extends ApplicationService {
-  final List<_BuiltinDesktopEntry> entries = [];
-  final Map<DesktopEntry, Widget> builders = {};
-
-  static const WindowEntry windowEntry = WindowEntry(
-    features: [
-      ResizeWindowFeature(),
-      SurfaceWindowFeature(),
-      FocusableWindowFeature(),
-      ToolbarWindowFeature(),
-    ],
-    layoutInfo: FreeformLayoutInfo(
-      position: Offset(32, 32),
-      size: Size(1280, 720),
-    ),
-    properties: {
-      ResizeWindowFeature.minSize: Size(480, 360),
-      SurfaceWindowFeature.elevation: 4.0,
-      SurfaceWindowFeature.shape: Constants.mediumShape,
-      SurfaceWindowFeature.background: PangolinWindowSurface(),
-      ToolbarWindowFeature.widget: PangolinWindowToolbar(
-        barColor: Colors.transparent,
-        textColor: Colors.black,
-      ),
-      ToolbarWindowFeature.size: 40.0,
-    },
-  );
-
```

Comment:
was this the result of a merge or similar?

Reply:
It relied on the built in apps which aren't part of the repo anymore

Reply:
leave the old impl and instead of relying on the old app list just rely on an empty list. the current built in impl could be useful for wm debugging